### ISSUE TITLE
Deprecated lifecycle method in redirect.js

### ISSUE THREAD

**Issue by arananegra** [mjrussell/redux-auth-wrapper#251]
Hi! In https://github.com/mjrussell/redux-auth-wrapper/blob/master/src/redirect.js there is a componentWillMount call which will be removed on next React versions. It will be great to update this file in order to make redux-auth-wrapper compatible with the upcoming React 17.x versions.

Thanks!

```
backend.js:6` Warning: componentWillMount has been renamed, and is not recommended for use. See https://fb.me/react-unsafe-component-lifecycles for details.

* Move code with side effects to componentDidMount, and set initial state in the constructor.
* Rename componentWillMount to UNSAFE_componentWillMount to suppress this warning in non-strict mode. In React 17.x, only the UNSAFE_ name will work. To rename all deprecated lifecycles to their new names, you can run `npx react-codemod rename-unsafe-lifecycles` in your project source folder.

Please update the following components: Redirect
```

**Comment by davidkhierl** [comment]
up for this one

**Comment by pavolgolias** [comment]
Is there any progress in this issue? After some digging in my project I found out that this lib creates warnings for me too.

**Comment by arananegra** [comment]
> Is there any progress in this issue? After some digging in my project I found out that this lib creates warnings for me too.

I don't know if @mjrussell is still maintaining this library. As I said, @JackHowa submitted a pull request addressing this warning issue but it doesn't seem he is tracking it.

**Comment by pavolgolias** [comment]
Yes I see it now. Lets hope this lib is still maintained, I like this lib :)

**Comment by mjrussell** [comment]
Fixed in #250

**Comment by mjrussell** [comment]
> I like this lib

Thanks 😄 sorry I haven't been the best maintainer...not doing much in React these days
